src/zapv2/core.py

```diff
@@ -693,3 +699,15 @@ def send_har_request(self, request, followredirects=None, apikey=''):
         if followredirects is not None:
             params['followRedirects'] = followredirects
         return (self.zap._request_other(self.zap.base_other + 'core/other/sendHarRequest/', params))
+
+    def file_download(self, filename, apikey=''):
+        """
+        Download a file from the transfer directory
+        """
+        return (self.zap._request_other(self.zap.base_other + 'core/other/fileDownload/', {'fileName': filename, 'apikey': apikey}))
+
+    def file_upload(self, filename, filecontents, apikey=''):
```

Comment:
Worth testing once to see if this works as expected...

Reply:
It doesn't :(

Reply:
Now using the expected HTTP method.

Reply:
Great, thank you!